hieradata/dev/wso2/wso2greg/5.2.0/default/store.yaml

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+classes:
+  - wso2greg
+wso2::pack_filename: "%{::product_name}-%{::product_version}.zip"
+wso2::pack_extracted_dir: "%{::product_name}-%{::product_version}"
+
+wso2::template_list :
+#        - repository/conf/datasources/social-datasources.xml
+        - repository/conf/identity/sso-idp-config.xml
+
+wso2::service_name: wso2greg
+wso2::hostname : wso2greg-store
+wso2::mgt_hostname : wso2greg-store
+
+# Host mapping to be made in etc/hosts
+wso2::hosts_mapping :
+  greg-node :
+    ip_address : 127.0.0.1
+    hostname : greg-node-2
+  localhost :
+    ip_address : 127.0.0.1
+    hostname : localhost
+
+wso2::file_list :
+#  - lib/org.wso2.carbon.server-4.2.0.jar
+#  - repository/components/lib/jackson-annotations-2.5.4.jar
+#  - repository/components/lib/jackson-core-2.5.4.jar
+#  - repository/components/lib/jackson-databind-2.5.4.jar
+#  - repository/components/lib/wso2ppaas-membership-scheme-4.1.0.zip
+#  - repository/components/lib/kubernetes-membership-scheme-1.0.0-SNAPSHOT.jar
```

Comment:
These jars are required only for Kubernetes platform.

repository/components/lib/jackson-annotations-2.5.4.jar
repository/components/lib/jackson-core-2.5.4.jar
repository/components/lib/jackson-databind-2.5.4.jar
repository/components/lib/kubernetes-membership-scheme-1.0.0-SNAPSHOT.jar

Since these profiles are made for default platform, you can remove these jars.